src/test/py/bazel/bzlmod/bazel_module_test.py

```diff
@@ -285,7 +285,12 @@ def testArchiveOverrideWithBadLabelPatch(self):
     )
     exit_code, _, stderr = self.RunBazel(['run', '//:main'], allow_failure=True)
     self.AssertNotExitCode(exit_code, 0, stderr)
-    self.assertIn("@@[unknown repo 'bbb' requested from @@]", '\n'.join(stderr))
+    self.assertIn(
+        "Error in <toplevel>: in call to 'http_archive' repo rule, no"
```

Comment:
The `in <toplevel>` part is pretty much just noise, can we drop it?

Reply:
Could we drop it when it is `<toplevel>`? It can also stay as is though, it's not confusing, just not really helpful.